GCMessengerSDKSample/GCMessengerSDKSample/ChatWrapperViewController.swift

```diff
@@ -146,13 +146,13 @@ extension ChatWrapperViewController: ChatControllerDelegate {
     }
     
     func requestNotificationAuth() {
```

Comment:
since the purpose of this function changed, I would change the naming to 'checkNotificationAuthStatus', something like that